### PR TITLE
Adding XCode 10 support

### PR DESCRIPTION
**Pull request by ldavisindg**
Copied your XCode 8 installer mostly verbatim, added the xcodebuild -runFirstLaunch command and added become_method: sudo on 'install additonal xcode components'

### CODE REVIEW

File: tasks/xcode_10.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  become_method: sudo
+  register: dev_tools_security_status
+
+- name: enable developer mode
+  command: DevToolsSecurity -enable
+  become: yes
+  become_method: sudo
+  when: "'disabled' in dev_tools_security_status.stdout"
+
+
```
Comment (nickcharlton) [inline tasks/xcode_10.yml]: Could we drop the final two blank spaces?

**Review by nickcharlton — COMMENTED** [review]
This is looking good! I'm going to have to manually test this first and I won't be able to that quite yet.

File: tasks/xcode_10.yml (hunk 1 above)
Comment (ldavisbna) [reply tasks/xcode_10.yml]: Forgot about this for a while, done.

**Comment by nickcharlton** [comment]
I never did get around to testing this, but now Xcode 10 is quite old and I trust you did actually do that. So I'm just going to merge it.

**Pull request merged**